### PR TITLE
Draft regression tutorial

### PR DESCRIPTION
**Pull request by r-tinn**

### CODE REVIEW

File: src/Tutorials/GausianProcessRegressor.cs [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        {
+            var x = data.Select(tup => tup.x);
+            var y = data.Select(tup => tup.y);
+
+            // Shift targets so mean is 0
+            var meanY = y.Sum() / y.Count();
+            y = y.Select(val => val - meanY);
+
+            // Scale data to lie between 1 and -1
+            var absoluteMaxY = y.Select(val => System.Math.Abs(val)).Max();
```
Comment (philipjball) [inline src/Tutorials/GausianProcessRegressor.cs:140]: Better to choose a more robust statistic; variance scaling is slightly more robust for example. A single max/min being an outlier will cause this approach to squash say 99% of the data into a small range and leave the 1% remaining data at the extreme -1/1.

Here's a guideline:
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.RobustScaler.html

**Comment by philipjball** [comment]
Shouldn't the PR be merged into `master`, not `pjball/data-set-generator`?

File: src/Tutorials/GausianProcessRegressor.cs (hunk 1 above)
Comment (r-tinn) [reply src/Tutorials/GausianProcessRegressor.cs:140]: Thanks mate I'll do that instead

**Comment by r-tinn** [comment]
> Shouldn't the PR be merged into `master`, not `pjball/data-set-generator`?

I was just going to submit this PR into infer.Net, was putting this PR up just to make it easy for you to look at

**Pull request closed without merging**